src/windows/CMakeLists.txt

```diff
@@ -1,7 +1,47 @@
+# The Broker Service is installed in one of two ways: from the Windows installer built in this
+# repository, or by invoking the service executable directly with the argument "-install".
+#
+# The installer cannot install the service by invoking the executable, because the service
+# installation must be part of the MSI table so that it can be reference-counted by dependent
+# products and cannot be removed when products still exist on the system that need it.
+#
+# The service configuration options, which are set here in both service_config.wxi and in
+# service_config.h, are defined below.
+
+set(BROKER_SERVICE_NAME "ETC RDMnet Broker")
+set(BROKER_SERVICE_DISPLAY_NAME "ETC RDMnet Broker")
+set(BROKER_SERVICE_DESCRIPTION "Implements an RDMnet message broker as defined in ANSI E1.33")
+
+# The following pairs must match; see https://wixtoolset.org/documentation/manual/v3/xsd/wix/serviceinstall.html
+# and https://docs.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-createservicew
+set(BROKER_SERVICE_C_START_TYPE SERVICE_AUTO_START)
+set(BROKER_SERVICE_WIX_START_TYPE "auto")
```

Comment:
So are you just having cmake create these are variables that both files can then consume? That seems pretty great.

Reply:
Yes, the variables are referenced by the `configure_file()` statements below.